front-end/src/components/BillingView.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class BillingView extends Component {
+  render() {
+    return (
+      <div>
+        <h1>Billing View</h1>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = () => ({});
```

Comment:
If you aren't using this, you don't need it. You should just be about to pass nothing or `null`